docker-storage-setup.1

```diff
@@ -98,36 +99,38 @@ WIPE_SIGNATURES:
       overwritten as suitable. This also means that if there is any
       data on disk, it will be lost.
 
-DOCKER_ROOT_VOLUME:
+ROOT_VOLUME:
       By default no new volume and filesystem will be setup
       for docker root dir. Docker creates /var/lib/docker/ on
       top of underlying filesystem for storing images and containers.
 
       To carve out a separate logical volume for storing docker
-      images/containers/volumes data, set DOCKER_ROOT_VOLUME=yes
+      images/containers/volumes data, set ROOT_VOLUME=yes
       NOTE: devicemapper stores images/containers on thin pool.
 
       docker-storage-setup creates a logical volume with an XFS
       filesystem mounted on docker root directory
       (default: /var/lib/docker).
+      NOTE: This option replaces DOCKER_ROOT_VOLUME which has been deprecated
 
-DOCKER_ROOT_VOLUME_SIZE:
+ROOT_VOLUME_SIZE:
       Specify the desired size for docker root volume. It defaults
       to 40% of all free space.
 
-      DOCKER_ROOT_VOLUME_SIZE can take values acceptable to
+      ROOT_VOLUME_SIZE can take values acceptable to
       "lvcreate -L" as well as some values acceptable to to 
       "lvcreate -l". If user intends to pass values acceptable 
       to "lvcreate -l", then only those values which contains "%" 
       in syntax are acceptable.  If value does not contain "%" it 
       is assumed value is suitable for "lvcreate -L".
 
       NOTE: If both STORAGE_DRIVER=devicemapper and 
-      DOCKER_ROOT_VOLUME=yes is set, docker-storage-setup would set
+      ROOT_VOLUME=yes is set, docker-storage-setup would set
       up the thin pool for devicemapper first, followed by docker 
```

Comment:
I really don't like renaming DOCKER_ROOT_VOLUME to ROOT_VOLUME because it is strongly tied to bind mounting lv on /var/lib/docker.

Can we decouple that first. How about create a new variable ROOT_VOLUME_MOUNT_DIR.  By default it will scan /etc/sysconfig/docker for -g otherwise use /var/lib/docker. Otherwise user can override it. Now ROOT_VOLUME will be mounted on ROOT_VOLUME_MOUNT_DIR.

@shishir-a412ed can you create a separate PR to take care of it.  Once this change is in, it decouples ROOT_VOLUME with mount point location and renaming DOCKER_ROOT_VOLUME to ROOT_VOLUME is much more reasonable.